README.md

```diff
@@ -50,6 +50,21 @@ Unpublish your post using:
 
     $ bundle exec jekyll unpublish _posts/2014-01-24-my-new-draft.md
 
+## Configuration
+
+To tune default plugin configuration edit `jekyll_compose` section of jekyll config file.
+
+```
+  jekyll_compose:
+    auto_open: true
+```
+
+and make sure that you have `EDITOR` or `JEKYLL_EDITOR` env variable set.
+
+The latter one will override default `EDITOR` value.
+
+It to open newly generated post in selected editor, `vim` in this case.
```

Comment:
Maybe: This will open a newly generated post in your selected editor.

I don't think we need to mention `vim`, as we're not using it as a default or anything.